scripts/add_procedures.sh

```diff
@@ -1,37 +1,86 @@
 #!/bin/bash
 
 #suppose l'existence du contenu de ../sql_templates dans /usr/local/bin/
-SCHEMA='public'
 
+SCHEMA="public"
+
+PG_HOST="localhost"
 PG_PORT=5432
 PG_USER="postgres"
+PGPASSWORD="postgres"
 DB_NAME="pgrouting"
 
-psql -v ON_ERROR_STOP=1 --username "$PG_USER" --dbname "postgres" <<-EOSQL
+CREATE_DBS=True
+
+TEMPLATES_DIR=/usr/local/bin
+WORK_DIR=/usr/local/bin
+
+while [ True ]; do
+    if [ "$1" = "--schema" -o "$1" = "-s" ]; then
+        SCHEMA=$2
+        shift 2
+    elif [ "$1" = "--hote" -o "$1" = "-h" ]; then
+        PG_HOST=$2
+        shift 2
+    elif [ "$1" = "--port" -o "$1" = "-p" ]; then
+        PG_PORT=$2
+        shift 2
+    elif [ "$1" = "--user" -o "$1" = "-u" ]; then
+        PG_USER=$2
+        shift 2
+    elif [ "$1" = "--dbname" -o "$1" = "-d" ]; then
+        DB_NAME=$2
+        shift 2
+    elif [ "$1" = "--no-create-dbs" -o "$1" = "-ndbs" ]; then
+        CREATE_DBS=False
+        shift 1
+    elif [ "$1" = "--templates-dir" -o "$1" = "-td" ]; then
+        TEMPLATES_DIR=$2
+        shift 2
+    elif [ "$1" = "--work-dir" -o "$1" = "-wd" ]; then
+        WORK_DIR=$2
+        shift 2    
+    elif [ "$1" = "--pass" -o "$1" = "-P" ]; then
+        PGPASSWORD=$2
+        shift 2
+    else
+        break
+    fi
+done
+
+export PGPASSWORD=$PGPASSWORD
+
+if [ "$CREATE_DBS" = True ] ;then
+
+echo "Creating database \"$DB_NAME\" and schema $SCHEMA..."
+
+psql -v ON_ERROR_STOP=1 --username "$PG_USER" --dbname "postgres" --host "$PG_HOST" --port $PG_PORT <<-EOSQL
```

Comment:
For this line and all the others psql commands, I have an error. 
```
psql: error: could not connect to server: Connection refused
pgrouting-server | 	Is the server running on host "localhost" (127.0.0.1) and accepting
pgrouting-server | 	TCP/IP connections on port 5432?
```
I think it's because there no more pg_hba.conf, but I'm not sure. 

To replicate this, you will have to remove your docker volume and run the image. If the volume exists, the run will not execute this bash script.

Reply:
I changed the default value of `PG_HOST` to `''`.
With this there is no connection error in docker.

I tried with the develop branch and when I add --host "localhost" I have the same issue.